_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,42 @@
+
+2018 also saw some very engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you would not otherwise have the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should feel comfortable doing, and we’ll continue to develop these events for the years to come.
+
+
+# Something New:
+
+There is only 8 months, 7 days, and 6 hours until check-in, but who’s counting? HackIllinois 2019 will be the 6th time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. We have established our mission this year:
+
+**_To create an environment conducive to education, collaboration, and mentorship within the Open Source Community_**
+
+We _create an enviroment_  in that our event is not defined by a simple "code for a prize" model. We plan to pack the entire event with content for attendees, from UIUC and over a dozen other universities, to have a unique experience throughout the weekend. HackIllinois carries development, social, and networking responsibilities, and these establish the three pillars —  _education, collaboration, and mentorship_ — of our event. The theme this year follows the trail that has been set forth by our past directors and staff, but we wanted to assert our post _within the Open Source community_. We fully believe that our event betters the Open Source Community: by bringing in new students, continuing development on projects we believe in, and showcasing the talent pool that we have to offer.
+
+More specifically, this year we want to make HackIllinois **exciting**. We are putting as much emphasis as we can on making this event more than 36 hours of coding; learning about open source is not enough, in 2019. It will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We will run interactive workshops, present lively mini-events, and host top industry talent for our weekend in February. The Open Source experience will be unique from the past: exciting to both the OSS gurus and those who have not yet explored this region of the technology world.
```

Comment:
"**exciting**" -> "even more **exciting**"
"learning about open source..." -> "simply learning about open source is not enough, especially in 2019"
Just ", in 2019" sounds kind of abrupt, the especially eases some of that.

Reply:
+1 on “more exciting” rather than just “exciting”. We don’t want the vibe to be “HackIllinois was boring in the past” or something like that